.env-dev

```diff
@@ -0,0 +1,23 @@
+NODE_ENV=production
+log_level=info
+PORT=8000
+approval_treshold=200
+office_manager=[slack id]
+news_channel=[channel id]
+orders_channel=[channel id]
+alza_username=@alza-username
+alza_password=@alza-password
+alza_cz_username=@alza-cz-username
+alza_cz_password=@alza-cz-password
+alza_hu_username=@alza-hu-username
+alza_hu_password=@alza-hu-password
+db_host=@alzabot-db-host
+db_port=5432
+db_user=alzabot
+db_password=@alzabot-db-password
+db_name=alzabot
+db_schema=alzabot
+slack_api_token=[slack_api_token]
+slack_bot_token=[slack_bot_token]
+airtable_api_key=[airtable_api_key]
+airtable_base=[airtable_base]
```

Comment:
Missing newline at the end of file. Configure your editor properly and you won't need to care about this in the future.